### PR TITLE
use new bundle strategy

### PR DESCRIPTION
**Pull request by jrushlow**
~wip -~ see https://github.com/symfony/symfony-docs/pull/19793

- [x] double check the structure against https://symfony.com/doc/current/bundles/best_practices.html && the PR mentioned above
- [x] test locally
- [x] test in linked app on linux
- [ ] test in linked app on macos
- [ ] test again...
- [x] ~add test to ensure if `lifetime` is changed in userland - the new value is used in the helper... _this may already be a thing_~ - To be completed under #187

### CODE REVIEW

**Comment by yceruto** [comment]
About the directory structure, there is one more possible change that matches the modern skeleton: the `translations/` directory can be moved to the root level too

File: src/SymfonyCastsVerifyEmailBundle.php [hunk 1]
```diff
@@ -9,22 +9,37 @@
...
-    public function getContainerExtension(): ?ExtensionInterface
+    protected string $extensionAlias = 'symfonycasts_verify_email';
+
+    public function configure(DefinitionConfigurator $definition): void
+    {
+        $definition->rootNode()
+            ->children()
+                ->integerNode('lifetime')
+                    ->defaultValue(3600)
+            ->info('The length of time in seconds that a signed URI is valid for after it is created.')
```
Comment (yceruto) [inline src/SymfonyCastsVerifyEmailBundle.php:31]: Is the line indent missing here?

**Comment by jrushlow** [comment]
> About the directory structure, there is one more possible change that matches the modern skeleton: the `translations/` directory can be moved to the root level too

@yceruto - I was just thinking about the best way to handle that. And I believe it'll be done outside of this PR. `reset-password` && `verify-email` have the exact same translations - to keep from adding too much noise to this PR - I need to `mv` and add any missing translations that exist in one bundle but not the other.

File: src/SymfonyCastsVerifyEmailBundle.php (hunk 1 above)
Comment (jrushlow) [reply src/SymfonyCastsVerifyEmailBundle.php:31]: note to self: check php-cs-fixer rules to see if its possible to flag this in the future...

**Comment by jrushlow** [comment]
@yceruto - I'm wondering if we could / should add array "shape" annotation(s) to https://github.com/symfony/symfony/blob/65ccca07588b108246a2aa3f5c8214e045c376be/src/Symfony/Component/HttpKernel/Bundle/AbstractBundle.php#L38 for better DX 

PHPStan Squawk:
image

Fixed by: https://github.com/SymfonyCasts/verify-email-bundle/pull/186/commits/089cec8d33e108a45bcccd9ad1ffd30fffc4c241

**Review by bocharsky-bw — APPROVED** [review]
Code looks good to me

**Pull request merged**